calculator33.py

```diff
@@ -1,26 +1,10 @@
-operations_count = 0
+def integer_overflow_noncompliant():
+    # Noncompliant: Number larger than limit of the datatype is stored.
+    arr = np.array([[100000000]], dtype=np.int8)
 
-def main():
-    ask_again = True
-    while(ask_again):
-        a = input("Enter the numerator: ")
-        b = input("Enter the denominator: ")
-        result = perform_division(a,b)
-        print(result)
-        ask_again = input("Do you want to perform another operation? Enter yes or no: ")
-        if(ask_again == 'yes'):
-            ask_again = True
-        else:
-            ask_again = False
-            print("You performed " + str(operations_count) + " operations, bye!")
 
-
-def perform_division(a,b):
-    global operations_count
-    try:
-        operations_count += 1
-        return int(a)/int(b)
-    except Exception as e:
-        pass
-
-main()
\ No newline at end of file
+def create_session_noncompliant():
+    import boto3
+    # Noncompliant: uses hardcoded secret access key.
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Your code uses hardcoded AWS credentials which might allow unauthorized users access to your AWS account. These attacks can occur a long time after the credentials are removed from the code. We recommend that you set AWS credentials with environment variables or an AWS profile instead. You should consider deleting the affected account or rotating the secret key and then monitoring Amazon CloudWatch for unexpected activity.
[https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html)